4.0.0/migration.md

```diff
@@ -166,24 +298,29 @@ Stopping the MySQL migration job after 10 hours indicates that the `mysql-latest
         deploy.go:1583] waiting mysql latest migrations job to finish... (attempt 5/3600)
         deploy.go:1583] waiting mysql latest migrations job to finish... (attempt 6/3600)
         deploy.go:1583] waiting mysql latest migrations job to finish... (attempt 7/3600)
+        ...
+        deploy.go:1583] waiting mysql latest migrations job to finish... (attempt 3600/3600)
+        deploy.go:1587] mysql-latest-migrations-job failed
+
 
+1. It’s also safe to stop the Installer with `CTRL + c` instead of waiting for 10 hours: the migration will be retried on the next run, as it's an idempotent operation.
 
-1. Stop the installer  with CTRL + c because the migration will be retried on the next run.
-2. Check which initialization failed. To do so, describe the pod related to the `mysql-latest-migrations` job.
+2. Check which MySQL migration failed. To do so, describe the pod related to the `mysql-latest-migrations` job:
```

Comment:
Check which MySQL migration is failed. To do so, describe the pod related to the `mysql-latest-migrations` job: